### PR TITLE
[cp] 3.17.2 cherrypicks

### PR DESCRIPTION
**Pull request by chris1984**

### CODE REVIEW

**Comment by chris1984** [comment]
[test katello]

**Review by jjeffers — APPROVED** [review]
APJ

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,52 @@
+# 3.17.2 (2021-02-09)
```
Comment (jturel) [inline CHANGELOG.md:1]: I see 3.17.2 twice in this changelog :)

Comment (chris1984) [reply CHANGELOG.md:1]: ok I will fix and open a new PR me--

Comment (chris1984) [reply CHANGELOG.md:1]: Fixed with https://github.com/Katello/katello/pull/9173

**Pull request merged**